pkg/scaling/scale_handler.go

```diff
@@ -440,7 +440,10 @@ func (h *scaleHandler) GetScaledObjectMetrics(ctx context.Context, scaledObjectN
 				}
 
 				if !metricsFoundInCache {
+					startTime := time.Now()
```

Comment:
could you please move the logic for measuring latency inside `cache.GetMetricsForScaler()` and return it from there?

Reply:
good suggestion, it will make the latency more accuracy, updated